maintainer/release.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Relying on `/bin/bash` is not so cool:

- Some people use other shells.
- It might not be at this path on all OSs.

```suggestion
#!/usr/bin/env sh
```